ansible/roles/alertmanager/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+go_arch: amd64
+
+alertmanager_user:   prometheus
+alertmanager_group:  prometheus
```

Comment:
Fair enough, I guess we can create a special user for alertmanager in that case.